pkg/controller/command/vdri/command.go

```diff
@@ -101,6 +136,132 @@ func (o *Command) CreatePublicDID(rw io.Writer, req io.Reader) command.Error {
 	return nil
 }
 
+// SaveDID saves the did doc to the store
+func (o *Command) SaveDID(rw io.Writer, req io.Reader) command.Error {
+	request := &DIDExt{}
```

Comment:
why it is called DIDExt?

Reply:
there is not any specific reason, i just saw used in the other command folder this word. I can change it to DIDArgs if its confusing